### PR TITLE
tools: add the doc of TiDB-Binlog Monitoring Metrics

### PR DESCRIPTION
**Pull request by CaitinChen**
Via https://github.com/pingcap/docs-cn/pull/946
@lilin90 @WangXiangUSTC PTAL

### CODE REVIEW

File: tools/tidb-binlog-monitor.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+### binlog_drainer_checkpoint_tso_no_change_for_1m
+
+- Description: Drainer `checkpoint` has not been updated for one minute
+- Monitoring rule: changes(binlog_drainer_checkpoint_tso[1m]) < 1
+- Solution: Check whether all the Pumps that are not offline are running normally
+
+### binlog_drainer_execute_duration_time_more_than_10s
+
+- Description: The transaction time it takes for Drainer to synchronize data to TiDB. If it is too large, the synchronization of Drainer is affected
```
Comment (lilin90) [inline tools/tidb-binlog-monitor.md]:
```suggestion
- Description: The transaction time it takes Drainer to synchronize data to TiDB. If it is too large, the Drainer synchronization of data is affected
```

File: tools/tidb-binlog-monitor.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+- Monitoring rule: histogram_quantile(0.9, rate(binlog_drainer_execute_duration_time_bucket[1m])) > 10
+- Solutions:
+    
+    - Check the TiDB cluster state
+    - Check the Drainer log or monitor. If a DDL operation causes this problem, just ignore it
```
Comment (lilin90) [inline tools/tidb-binlog-monitor.md]:
```suggestion
    - Check the Drainer log or monitor. If a DDL operation causes this problem, you can ignore it
```

**Comment by CaitinChen** [comment]
@lilin90 PTAL

File: tools/tidb-binlog-monitor.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
+---
+title: TiDB-Binlog Monitoring Metrics
+summary: This document introduces three levels of monitoring metrics of TiDB-Binlog.
```
Comment (lilin90) [inline tools/tidb-binlog-monitor.md]: Please refer to other documents for the summary sentence pattern. Only several specific sentence patterns are used.

**Pull request merged**